salt/elife-bot/config/opt-elife-bot-digest.cfg

```diff
@@ -26,6 +26,7 @@ iiif_info_url:
 [elife]
 medium_license: cc-40-by
 medium_image_url: {{ pillar.elife_bot.iiif.url }}digests/{msid:0>5}%2F{file_name}/full/full/0/default.jpg
+medium_footer_pattern: <hr/><p><em>Originally published at <a href="{{ pillar.elife_bot.journal.url }}/digests/{msid:0>5}">{{ pillar.elife_bot.journal.url }}/digests/{msid:0>5}</a>.</em></p>
```

Comment:
> {msid:0>5}

what is that?

Reply:
It's an int/str safe 5 digit zero fill for `.format()`

Reply:
is it jinja or part of another templating language?

Reply:
Just `.format`, like used here https://github.com/elifesciences/digest-parser/blob/develop/digestparser/medium_post.py#L22 , following https://docs.python.org/3/library/string.html#formatspec

Reply:
so, elife-bot reads digest.cfg and then interpolates that string. It feels like a real jumble.

Reply:
The content itself could maybe be upgraded to a template file, other parts are individual data points. This project may have overused the `.cfg` file concept.